index.js

```diff
@@ -61,6 +61,7 @@ module.exports = class PearUpdater extends ReadyResource {
     this.snapshot = null
     this.updated = false
     this.updating = false
+    this._updateTarget = null
```

Comment:
why is this needed?

Reply:
This is so that it stops updating after it reaches the next closest unskippable update (until pear shuts down to apply the update by calling applyUpdate)

Tried using just checkout but after update() finishes and checkout is now the same as the next unskippable update, it would keep updating to the next unskippable update even without applyUpdate being called.

Reply:
i see, sounds like we shoudl make a that a boolean flag instead

Reply:
ie `.frozen`

Reply:
which makes it not update again, you can also call it `.unskippable` boolean